week-03/main.js

```diff
@@ -43,10 +53,42 @@ var list = [1, 2, 3, 4, 5];
 
 var list = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
 
+var oddOnly = function () {
+  return list.filter(function (n){
+    return n % 2 !== 0;
+  });
+};
+
+var doubleOdds = oddOnly(list).map(function (n){
```

Comment:
You're passing in `list`, but in the function above you didn't define it to take a parameter. It's still working because `list` is global. 

You could just write the code above like the two below and assign the return value to a variable.

```
var evenList = list.filter(function (n){
  return n % 2 !== 0;
});
```